tests/displayAllGamesTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class displayAllGamesTest extends TestCase {
+
+    public function testDisplayGamesOutput(): void
```

Comment:
```suggestion
    public function testDisplayGamesOutput_Success(): void
```
Including if this is a success, failure or malformed test in the function name would be standard